src/orca-jedi/state/State.h

```diff
@@ -105,6 +105,8 @@ class State : public util::Printable,
   const oops::Variables & variables() const {return vars_;}
   oops::Variables & variables() {return vars_;}
 
+  atlas::Field getField(int) const;
```

Comment:
Sorry, I just realised one more thing - this one won't be tested. Could you please add a test to `src/tests/orca-jedi/test_state.cc` just to retrieve a field and check its name matches what is expected?  I can help with this if needed.

Reply:
I'll give it a go and let you know if I'm stumped.